kuma/events/tests/test_views.py

```diff
@@ -0,0 +1,9 @@
+from kuma.core.urlresolvers import reverse
+
+
+def test_events(client, db):
```

Comment:
I don't think we should bother testing this (it basically tests ``shared_cache_control`` and ``RedirectView``, which are tested elsewhere), and I really don't think we should create a new app just so we can.